devops/dependencies.json

```diff
@@ -19,9 +19,9 @@
       "root": "{DEPS_ROOT}/opencl/runtime/linux/oclgpu"
     },
     "level_zero": {
-      "github_tag": "v1.14.0",
-      "version": "v1.14.0",
-      "url": "https://github.com/oneapi-src/level-zero/releases/tag/v1.14.0",
+      "github_tag": "v1.15.1",
+      "version": "v1.15.1",
+      "url": "https://github.com/oneapi-src/level-zero/releases/tag/v1.15.1",
```

Comment:
@intel/dpcpp-devops-reviewers, shouldn't this dependency be managed by unified runtime now? Updating Level Zero w/o UR might cause regressions.

Reply:
I've discussed that with @jandres742 and we'll keep the current scheme for now. L0 is supposed to be backward compatible so it should be ok to update it on its own.

Making UR/L0 updates in a lockstep is problematic because one is done via image rebuilding and another is SYCL build-time. We currently have a small race condition when `dependencies.json` has been updated in `origin/sycl` but images are still being rebuilt and another testing starts. It's mostly problem-free now, but it will always result in an issue with lockstep updates. As such, L0 will always have to be updated first and keeping the current automation might save some manual labor while being relatively safe (modulo unexpected abi-breaks in L0).

Reply:
thanks @aelovikov-intel !